src/item.cpp

```diff
@@ -1439,6 +1487,29 @@ std::string item::info(bool showtext, std::vector<iteminfo> *dump, bool debug) c
                 }
             }
         }
+
+        // display the contents of what's inside the bag, if you have opened it before
+        if(is_item_container() && !is_container_empty()) {
+            // is it new to us?
+            if(!has_been_opened) {
+                dump->push_back(iteminfo("DESCRIPTION", _("This feels like there might be more inside.")));
+            // nope? list contents
+            } else {
+                //[davek] TODO: categorize contents of items
+                temp1.str("");
+                bool pizza = false;     // emulate above for commas
+                for(auto &elem : contents) {
+                    if(pizza) { temp1 << _(", "); }
+                    pizza = true;       // yum!
+                    nc_color c = elem.color_in_inventory();
+                    bool did_coloring = (c == c_ltgray) ? false : true;
+                    if(did_coloring) { temp1 << "<color_" << string_from_color(c) << ">"; }
+                    temp1 << item::nname(elem.type->id);
+                    if(did_coloring) { temp1 << "</color>"; }
```

Comment:
Why don't you print color tags for ltgray? This seems unnecessarily and potentially wrong (you don't know which default color is used to display the text).